scss/main.scss

```diff
@@ -225,3 +225,31 @@ h3 {
         display: inline;
     }
 }
+
+.page-header {
```

Comment:
I think most of this css can be replaced with the bootstrap classes probably

Reply:
so the problem is that the site currently depends on bootstrap 3, which doesn't support the `d-flex` etc. classes. i can update it to bootstrap 4, this however causes an error when being compiled with `pysassc` (thus, we might have to switch to another compiler).

let me know if i should do this update, for now i renamed the classes to match the naming of the needed bootstrap 4 classes.

for now i renamed the flexbox classes to match the naming of the bootstrap 4 ones.

Reply:
it isn't a problem with the compiler, there's just a bunch of breaking changes between 3 and 4 (I tried migrating at some point).  bs4 works fine with `pysassc` (it's the same as node-sass) -- I maintain it btw

Reply:
ah, okay, didn't look at the error message in detail tbh (as it's also possible with bs3). the error was reported in line 2 which is the bootstrap import, so i guessed the error was caused by bootstrap itself (but it's not).